src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -52,10 +52,10 @@ export const EventModal: React.FC<EventModalProps> = ({
   const goToEventLocation = () => {
     onHide();
 
-    if (event.room) {
-      enterRoom();
-    } else {
+    if (event.venueId) {
```

Comment:
Sorry, the logic here might wrong and could fail if the event is actually stored at the room-level. I haven't checked that use case, so definitely worth testing there.

Reply:
@margulies what was wrong with the existing logic? If there is a room, enter the room, otherwise enter the venue? That seems correct and straightforward to me? Not sure what the changes you're making here are for, but it doesn't look right to me?

Reply:
Additionally, `enterRoom` is defined in `src/hooks/useRoom.ts` and basically just ends up calling either `enterVenue` or `enterExternalRoom` depending on whether `roomVenue` exists.

Reply:
I know it looks weird, but if you try the OHBM setup, you'll see that clicking on a room location from the event modal returns: `//`
At least in the case of our current env/ohbm setup, where all the events are stored on the venue-level, the fix I suggested addressed this.

Reply:
Have we figured out the ‘why’ of that occurring? It’s better to know why than to make arbitrary changes that seem to fix the issue we haven’t even properly defined yet.